<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsAttributesFormProperties</class>
 <widget class="QWidget" name="QgsAttributesFormProperties">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QToolButton" name="mRemoveTabOrGroupButton">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../../images/images.qrc">
       <normaloff>:/images/themes/default/symbologyRemove.svg</normaloff>:/images/themes/default/symbologyRemove.svg</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="mAddTabOrGroupButton">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../../images/images.qrc">
       <normaloff>:/images/themes/default/symbologyAdd.svg</normaloff>:/images/themes/default/symbologyAdd.svg</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="mDragListWidget" native="true"/>
     <widget class="QWidget" name="mDropListWidget" native="true"/>
     <widget class="QWidget" name="mOptionsWidget" native="true">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="Testlable">
         <property name="text">
          <string>Widget Properties</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../images/images.qrc"/>
 </resources>
 <connections/>
</ui>
